app/utils/utils.py

```diff
@@ -0,0 +1,7 @@
+class Utils:
+    def treat_phone(phone_str: str) -> str:
+        new_phone = ""
+        for i in phone_str:
+            if i >="0" and i <="9":
```

Comment:
esse código pode ser mais "pythonico" se você fizer dessa forma:

```suggestion
            if "0" <= i <= "9":
```

segue o artigo de leitura para entender o que é um código pythonico [Como deixar seu código mais pythonico](https://medium.com/loftbr/9-formas-de-deixar-seu-c%C3%B3digo-mais-pythonico-526130e291cf)